public/javascripts/SVLabel/css/svl.css

```diff
@@ -8,10 +8,10 @@
 text {
     visibility: hidden;
 }
-
+/*
```

Comment:
The issue is there is a handful of elements in the audit page that uses the "visible" class, including the sound button and the curb ramp statistics on the tool's RHS. If I keep the class' css defined initially, those elements will show. I add in the css for the visible class once the data is loaded as per `$(".visible").css({"visibility": "visible"});` in Main.js. So, the class has no defined css initially, but once the page is loaded it does.